v2/dtos/responses/event.go

```diff
@@ -37,10 +37,10 @@ type MultiEventsResponse struct {
 
 // UpdateEventPushedByChecksumResponse defines the Response Content for PUT event as pushed DTO.
 // This object and its properties correspond to the UpdateEventPushedByChecksumResponse object in the APIv2 specification:
-// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/UpdateEventPushedByChecksumResponse
-type UpdateEventPushedByChecksumResponse struct {
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/UpdateEventPushedByIdResponse
```

Comment:
the name in line 38 should also be updated

Reply:
Updated.